### PR TITLE
[BUGFIX] Fixes bug allowing player to multiclick WG

### PR DESCRIPTION
**Pull request by ZomgPonies**
No more being both Owlman AND Griffon at the same time and removing a slot from other people to find.

### CODE REVIEW

File: code/game/machinery/wishgranter.dm [hunk 1]
```diff
@@ -24,22 +25,30 @@
 
 	else if(is_special_character(user))
 		user << "Even to a heart as dark as yours, you know nothing good will come of this.  Something instinctual makes you pull away."
+		return
+
+	else if(inuse)
```
Comment (tigercat2000) [inline code/game/machinery/wishgranter.dm]: This else-if chain is completely pointless since all of them just kill the proc.

Comment (Tastyfish) [inline code/game/machinery/wishgranter.dm]: Alternatively, most of the returns in this proc are pointless.

Comment (tigercat2000) [inline code/game/machinery/wishgranter.dm]: yeah it can work either way, but generally returns are cleaner

**Comment by Plotron** [comment]
Dayum son, I'm gonna miss the Owling Griffin.

**Pull request merged**